src/components/crowdsale/utils.js

```diff
@@ -130,13 +139,19 @@ export let getCrowdsaleData = async (initCrowdsaleContract, execID) => {
 
     if (crowdsaleStore.isDutchAuction) {
       const crowdsaleStatus = await getCrowdsaleStatus(...params).call()
-      const { max_sellable } = await isCrowdsaleFull(...params).call()
+      crowdsaleStatus.start_rate = crowdsaleStatus.start_rate || crowdsaleStatus[0]
+      crowdsaleStatus.end_rate = crowdsaleStatus.end_rate || crowdsaleStatus[1]
+      crowdsaleStatus.current_rate = crowdsaleStatus.current_rate || crowdsaleStatus[2]
+      crowdsaleStatus.time_remaining = crowdsaleStatus.time_remaining || crowdsaleStatus[4]
+      const tokens_remaining = crowdsaleStatus.tokens_remaining || crowdsaleStatus[5]
+      const _isCrowdsaleFull = await isCrowdsaleFull(...params).call()
+      const max_sellable = _isCrowdsaleFull.max_sellable || _isCrowdsaleFull[1]
```

Comment:
I use the array lookup because Proxy getters return only arrays. In the line, that you provided, array lookup wasn't made because it was made earlier in this line:
https://github.com/poanetwork/token-wizard/blob/4ec3c88c2f90b1f33fe2435bc93844ccca177dfb/src/components/crowdsale/utils.js#L98